docs/pages/middleware/images-worker.mdx

```diff
@@ -0,0 +1,88 @@
+# Images Worker
+
+The images worker serializes JSX images in frame definition responses into a URL which is rendered in a separate request and not bound by data URL size limits.
+
+It does this by serializing the JSX component tree into a string which is passed to the image worker route via a query param in a URL. The worker deserializes the JSX and renders it into a PNG image as a separate response.
+
+## Usage
+
+:::steps
+
+### Set up the images worker endpoint
+
+Optionally specify a secret to secure the endpoint.
+
+```tsx [./images/route.tsx]
+import { createImagesWorker } from "frames.js/middleware/images-worker/next";
+
+const imagesRoute = createImagesWorker({
+  secret: "SOME_SECRET_VALUE",
+});
+
+export const GET = imagesRoute();
+```
+
+### Add the `imagesWorkerMiddleware` middleware to your Frames app
+
+```ts [./frames/frames.ts]
+import { createFrames } from "frames.js/next";
+import { imagesWorkerMiddleware } from "frames.js/middleware/images-worker";  // [!code focus]
+
+export const frames = createFrames({
+  // ...
+  middleware: [ // [!code focus]
+    ImagesWorker.imagesWorker({ // [!code focus]
```

Comment:
is this part correct? Where does `ImagesWorker` come from?

Reply:
typo, fixed